pkg/operator/discover/discover.go

```diff
@@ -42,6 +43,9 @@ const (
 	discoverDaemonsetName             = "rook-discover"
 	discoverDaemonsetTolerationEnv    = "DISCOVER_TOLERATION"
 	discoverDaemonsetTolerationKeyEnv = "DISCOVER_TOLERATION_KEY"
+	deviceInUseCMName                 = "local-device-in-use-cluster-%s-node-%s"
```

Comment:
Can we add by which osd too?

Reply:
this name is also used by backends other than ceph